xml/admin_saltcluster.xml

```diff
@@ -372,6 +372,14 @@ block_wal_size: '5G'  # (optional, unit suffixes permitted)
 block_db_size: '5G'   # (optional, unit suffixes permitted)
 encrypted: true       # 'True' or 'False' (defaults to 'False')
 </screen>
+     <note>
+       <para>
+         When applying &drvgrps; in &productname; 7, the option previously 
+         called "encryption" in &deepsea; has been renamed to "encrypted".
+         Ensure you use this terminology when applying your service specification
+         otherwise the application will fail.
```

Comment:
The phrasing of this paragraph is a little weird. To me, it's asking to be rephrased, perhaps like this?

    The option previously called "encryption" in DeepSea has been renamed to "encrypted".
    When applying Drive Groups in SES7, ensure you use this new terminology in your service
    spec, otherwise the "ceph orch apply" operation will fail.

Reply:
I agree with @smithfarm here.